packages/ds-sam-sdk/test/__snapshots__/constraints.test.ts.snap

```diff
@@ -1,102 +1,102 @@
+country-aso-validator-7, inflation pmpe: 0.19999999999999998, mev pmpe: 0.1684210526315789, bid pmpe: 1, total pmpe: 1.3684210526315788, mnde_target_stake: 0, sam_target_stake: 75000, last constraint: ASO (dummy-aso)
+country-aso-validator-8, inflation pmpe: 0.19999999999999998, mev pmpe: 0.1684210526315789, bid pmpe: 1, total pmpe: 1.3684210526315788, mnde_target_stake: 0, sam_target_stake: 75000, last constraint: ASO (dummy-aso)
+country-aso-validator-9, inflation pmpe: 0.19999999999999998, mev pmpe: 0.1684210526315789, bid pmpe: 1, total pmpe: 1.3684210526315788, mnde_target_stake: 0, sam_target_stake: 75000, last constraint: ASO (dummy-aso)
+aso-validator-0, inflation pmpe: 0.19999999999999998, mev pmpe: 0.1684210526315789, bid pmpe: 1, total pmpe: 1.3684210526315788, mnde_target_stake: 0, sam_target_stake: 75000, last constraint: ASO (dummy-aso)
+aso-validator-1, inflation pmpe: 0.19999999999999998, mev pmpe: 0.1684210526315789, bid pmpe: 1, total pmpe: 1.3684210526315788, mnde_target_stake: 0, sam_target_stake: 75000, last constraint: ASO (dummy-aso)
+aso-validator-2, inflation pmpe: 0.19999999999999998, mev pmpe: 0.1684210526315789, bid pmpe: 1, total pmpe: 1.3684210526315788, mnde_target_stake: 0, sam_target_stake: 75000, last constraint: ASO (dummy-aso)
+aso-validator-3, inflation pmpe: 0.19999999999999998, mev pmpe: 0.1684210526315789, bid pmpe: 1, total pmpe: 1.3684210526315788, mnde_target_stake: 0, sam_target_stake: 75000, last constraint: ASO (dummy-aso)
+aso-validator-4, inflation pmpe: 0.19999999999999998, mev pmpe: 0.1684210526315789, bid pmpe: 1, total pmpe: 1.3684210526315788, mnde_target_stake: 0, sam_target_stake: 75000, last constraint: ASO (dummy-aso)
+aso-validator-5, inflation pmpe: 0.19999999999999998, mev pmpe: 0.1684210526315789, bid pmpe: 1, total pmpe: 1.3684210526315788, mnde_target_stake: 0, sam_target_stake: 75000, last constraint: ASO (dummy-aso)
+aso-validator-6, inflation pmpe: 0.19999999999999998, mev pmpe: 0.1684210526315789, bid pmpe: 1, total pmpe: 1.3684210526315788, mnde_target_stake: 0, sam_target_stake: 75000, last constraint: ASO (dummy-aso)
+aso-validator-7, inflation pmpe: 0.19999999999999998, mev pmpe: 0.1684210526315789, bid pmpe: 1, total pmpe: 1.3684210526315788, mnde_target_stake: 0, sam_target_stake: 75000, last constraint: ASO (dummy-aso)
+aso-validator-8, inflation pmpe: 0.19999999999999998, mev pmpe: 0.1684210526315789, bid pmpe: 1, total pmpe: 1.3684210526315788, mnde_target_stake: 0, sam_target_stake: 75000, last constraint: ASO (dummy-aso)
+aso-validator-9, inflation pmpe: 0.19999999999999998, mev pmpe: 0.1684210526315789, bid pmpe: 1, total pmpe: 1.3684210526315788, mnde_target_stake: 0, sam_target_stake: 75000, last constraint: ASO (dummy-aso)
+vote-acc--0000000000, inflation pmpe: 0.19999999999999998, mev pmpe: 0.1684210526315789, bid pmpe: 0, total pmpe: 0.36842105263157887, mnde_target_stake: 0, sam_target_stake: 150000, last constraint: BOND (vote-acc--0000000000)
```

Comment:
Validators starting with this one should be capped by the `VALIDATOR` constraint, in the new version they're capped by `BOND`. If you have trouble with finding the right setup of the test (increasing the bond amounts of the validators should probably be enough) then you can merge I guess (to not block the release), but this test now verifies `COUNTRY` and `ASO` constraints only, not also `VALIDATOR` like previously.

Ideally, you should add a new test to verify the logic of the cap increase specifically.